hack/build.sh

```diff
@@ -25,6 +25,11 @@ TAGS="${TAGS:-}"
 OUTPUT="${OUTPUT:-bin/openshift-install}"
 export CGO_ENABLED=0
 
+if [ ! -x data/data/bin/terraform ]
+then
+	(. hack/get-terraform.sh)
```

Comment:
Why source this inatead of `hack/get-terraform.sh`?

Reply:
Because I'm bad at shell ;)